core/src/sumeragi/message.rs

```diff
@@ -169,7 +146,7 @@ impl From<VersionedCommittedBlock> for BlockCommitted {
 #[non_exhaustive]
 pub struct BlockSyncUpdate {
     /// The corresponding block.
-    block: VersionedCommittedBlock,
+    pub block: VersionedCommittedBlock,
```

Comment:
This field was private on purpose, to always be sure that block will be revalidated before usage. 
However i think we can return to this in separate PR.

Reply:
Created #3492.